packages/react/src/components/DataGrid/DataList.tsx

```diff
@@ -2,5 +2,15 @@ import React from "react";
 import { BaseProps, DataGrid, Row } from ":/components/DataGrid/index";
 
 export const DataList = <T extends Row>({ rows, ...props }: BaseProps<T>) => {
-  return <DataGrid {...props} displayHeader={false} rows={rows} />;
+  return (
+    <DataGrid
+      {...props}
+      displayHeader={false}
+      rows={rows}
+      columns={props.columns.map((column) => ({
+        ...column,
+        enableSorting: false,
+      }))}
```

Comment:
nit: should we not disable sorting at the table level?
(cf. [here](https://github.com/numerique-gouv/lasuite-landingpage/pull/22))

Reply:
We could, be it hasn't be implemented by Cunningham, at the moment it is only handled at column level, I might create another PR for it 🤔